roles/openshift_openstack/templates/heat_stack.yaml.j2

```diff
@@ -1093,9 +1094,11 @@ resources:
             - { get_resource: flat-secgrp }
 {% else %}
             - { get_resource: node-secgrp }
+{% if openshift_openstack_num_cns > 0 %}
             - { get_resource: cns-secgrp }
+{% endif %}
 {% if openshift_use_kuryr|default(false)|bool %}
-          pod_secgrp:
+         pod_secgrp:
```

Comment:
Thanks for the PR! Quick question - is this space removal intentional? The original indent looks correct to me... ?

Reply:
ah, yep. thats a typo